docs/source/explore/get-started/install.rst

```diff
@@ -93,28 +93,28 @@ Windows Clean Install
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.  
-2. Uninstall any versions of Project Athena that are visible (Including any Steam installs). 
-3. Once Project Athena is uninstalled, browse to your %Program Files% directory. Delete all folders related to Project Athena. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``. 
+2. Uninstall any versions of Vircadia that are visible (Including any Steam installs). 
+3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``. 
 
 .. warning:: 
 
-    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Project Athena/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Project Athena/assignment-client. 
+    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Vircadia/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Vircadia/assignment-client. 
```

Comment:
These directory renamings haven't been done, yet. What to do? Make this and related changes in the docs in anticipation of it being updated in K3?

Reply:
I'll leave it to be correct and we'll update it come K3, I suppose.

Reply:
By that I mean, I'll leave it as Athena.